stubs/docutils/docutils/writers/latex2e/__init__.pyi

```diff
@@ -286,10 +286,10 @@ class LaTeXTranslator(nodes.NodeVisitor):
     def visit_docinfo(self, node: nodes.docinfo) -> None: ...
     def depart_docinfo(self, node: nodes.docinfo) -> None: ...
     @overload
-    def visit_docinfo_item(self, node, name: None = None) -> None: ...
+    def visit_docinfo_item(self, node) -> None: ...
     @overload
     @deprecated("The `name` parameter is deprecated and will be removed in Docutils 0.24.")
-    def visit_docinfo_item(self, node, name: str = ...) -> None: ...
+    def visit_docinfo_item(self, node, name: str | None = None) -> None: ...
```

Comment:
I think we only want type checkers to select this overload if an argument has actually been passed into the `name` parameter, so maybe this?

```suggestion
    def visit_docinfo_item(self, node, name: str | None) -> None: ...
```